Tasks/MobileCenterDistribute/mobilecenterdistribute.ts

```diff
@@ -406,7 +406,7 @@ async function run() {
         let app = utils.resolveSinglePath(appFilePattern);
         tl.checkPath(app, "Binary file");
 
-        let continueIfSymbolsNotFoundVariable = tl.getVariable('VSMobileCenterUpload.ContinueIfSymbolsNotFound');
+        let continueIfSymbolsNotFoundVariable = tl.getVariable('MobileCenterDistribute.ContinueIfSymbolsNotFound');
```

Comment:
It's this line that I'm most concerned about with upgrade scenarios.  Should we bump the major version due to this?

(TBQH, I'm not sure why this is a variable and not a task setting.)